ArduCopter/Parameters.cpp

```diff
@@ -1168,12 +1168,48 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
 
     // ID 60 is reserved for the SHIP_OPS
 
+    // extend to a new group
+    AP_SUBGROUPEXTENSION("", 61, ParametersG2, var_info2),
+
     // ID 62 is reserved for the SHOW_... parameters from the Skybrush fork at
     // https://github.com/skybrush-io/ardupilot
 
     AP_GROUPEND
 };
 
+/*
+  extension to g2 parameters
+ */
+const AP_Param::GroupInfo ParametersG2::var_info2[] = {
+
+    // @Param: PLDP_THRESH
+    // @DisplayName: Payload place thrust ratio to detect payload touchdown
+    // @Description: Ratio of vertical thrust during decent below which payload touchdown will trigger.
+    // @Range: 0.5 0.9
+    // @User: Standard
+    AP_GROUPINFO("PLDP_THRESH", 1, ParametersG2, pldp_throttle_placed_fraction, 0.9),
+
+    // @Param: PLDP_RNG_MIN
+    // @DisplayName: Minimum range finder altitude in m to trigger payload touchdown
+    // @Description: Minimum range finder altitude in m to trigger payload touchdown, set to zero to dissable.
+    // @Units: m
+    // @Range: 0 100
+    // @User: Standard
+    AP_GROUPINFO("PLDP_RNG_MIN", 2, ParametersG2, pldp_range_finder_minimum, 0.0),
+
+    // @Param: PLDP_DELAY
+    // @DisplayName: Minimum range finder altitude in m to trigger payload touchdown
```

Comment:
I think the display name and description are copy-pasted from the parameter above?